### PR TITLE
Implement author's profile page

### LINKED ISSUE #42: List post by user
We should have the ability to click on the author's name or avatar to list all posts by this author.

### PR DESCRIPTION
**Pull request by huytd**
Squash before merging.

Fixed #42 

image

Test build: https://webuild-blog-5r4ykdlvt-kingbazoka.vercel.app/page/1

### CODE REVIEW

File: pages/author/[host].tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+export const getStaticProps: GetStaticProps = async (context) => {
+  const { host } = context.params ?? {};
+  const parser = new Parser();
+  const authorId = channelsData.channels.findIndex(
+    (channel) => getHostName(channel.url) === host
+  );
+  const author = channelsData.channels[authorId] ?? null;
+  let docs: Doc[] = [];
+  if (author) {
+    const author = channelsData.channels[authorId];
```
Comment (lqt93) [inline pages/author/[host].tsx:33]: Duplicated with line 30

**Review by lqt93 — CHANGES_REQUESTED** [review]

File: pages/author/[host].tsx (hunk 1 above)
Comment (huytd) [reply pages/author/[host].tsx]: bingo 😆

File: pages/read/[encoded].tsx [hunk 2]
```diff
@@ -20,27 +20,34 @@ import { Author } from '@/types/sharedTypes';
...
+        article: {
+          ...article,
+          link: url,
+          content: DOMPurify.sanitize(article?.content ?? '')
+        },
+        author: matchedAuthor
+      }
+    };
+  } catch (error) {
+    console.error(error);
```
Comment (lqt93) [inline pages/read/[encoded].tsx:46]: Should we log the error :-?

**Review by lqt93 — COMMENTED** [review]
Instead of the console.log, the others LGTM. 💯

**Review by lqt93 — APPROVED** [review]

File: pages/read/[encoded].tsx (hunk 2 above)
Comment (huytd) [reply pages/read/[encoded].tsx:46]: Yes, for better debug

**Pull request merged**